src/main/tray.js

```diff
@@ -0,0 +1,163 @@
+/*!
+GPII Application
```

Comment:
Please update all these file comments to reflect file contents plus a short description of file's purpose